src/server/pkg/storage/chunk/limit_objclient.go

```diff
@@ -0,0 +1,74 @@
+package chunk
+
+import (
+	"context"
+	io "io"
+	"math"
+
+	"github.com/pachyderm/pachyderm/src/server/pkg/obj"
+	"golang.org/x/sync/semaphore"
+)
+
+var _ obj.Client = limitedObjClient{}
```

Comment:
Is this checking if `limitedObjClient` satisfies the `obj.Client` interface?

Reply:
yeah